operator/v1/types_dns.go

```diff
@@ -287,6 +287,17 @@ type ForwardPlugin struct {
 	//
 	// +optional
 	TransportConfig DNSTransportConfig `json:"transportConfig,omitempty"`
+
+	// protocolPolicy configures the protocol policy for upstream DNS requests.
+	//
+	// This field allows a cluster admin to optionally configure:
+	// * None configures CoreDNS for to use its default upstream protocol selection.
+	// * ForceTCP configures CoreDNS to use TCP for all upstream DNS requests, even if the client request uses UDP.
```

Comment:
Bulleted lists don't render well in the generated CRD.  It is necessary to note if we may change the default behavior in the future.  Often an API field has a "no opinion" option, and "None" seems like such an option; we usually use the empty string for "no opinion"/default values.  The following phrasing is more consistent with current API conventions and recent API changes:
```suggestion
	// protocolPolicy specifies the protocol to use for upstream DNS
	// requests.  Valid values for protocolPolicy are ForceTCP or omitted.
	// When omitted, that means no opinion, and the platform uses a default
	// behavior, which is subject to change over time.  Currently, the
	// default behavior is to use the protocol of the original client
	// request as the upstream protocol.  ForceTCP specifies that the
	// platform should use TCP for all upstream DNS requests, even if the
	// client request uses UDP.

```